sem_covid/entrypoints/notebooks/topic_modeling/pwdb.ipynb

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
<p>agree, I should find another solution. It was the fastest I could implement at the moment.</p><p> </p><p>I suppose technical words are so frequently met because of <code>str(someDataFrame)</code>. Those words are converted in string as well. </p>

Reply:
I would also check if they appear in the elasticsearch index.

Reply:
<p>It appears. That's why I reset the index, but it was dummy from me to think that it will change the situation with word frequency</p>